### ISSUE TITLE
"Potentially disturbing content" category

### ISSUE THREAD

**Issue by nikkehtine** [ajayyy/SponsorBlock#1096]
I'd like to ask for the potentially disturbing content category, to indicate graphic content, such as pictures that contain blood, or other content depicting (visually or with words) things that someone might find disgusting, repulsive, offensive, etc.

A possible video where this skippable section category might be useful is an informative video discussing a mild topic, that at some point shows disturbing pictures, with a clear warning from the creator ([example](https://youtu.be/g5ZUnQGFPpk?t=287))

Of course it wouldn't make sense to flag content in videos that are about potentially disturbing things, so there have to be some kind of guidelines to be followed.

Examples of videos that **shouldn't** be tagged with these segments:
- thriller/horror videos
- creepypasta readings
- stories (real or fictional) of murders, serial killers, mysterious pictures, disappearances of people, things that might seem supernatural
- informative videos about health and safety, such as first aid instructional videos, guides on how to stay safe on road
- videos discussing medicine, such as what happens to the organism if you drink X or inject Y, descriptions/depictions of diseases, visualizations of the human body or certain organs
- video reviews/discussions of violent or thriller media: movies, video games, books, paintings
- news programs

Given the nature of the content that would fall under this category I also suggest setting it by default to "Skip manually". This would give the viewer an option to skip that segment without forcibly skipping it by default for those who haven't changed the extension's settings but wouldn't want to skip it.

**Comment by aphix** [comment]
+1 for specifically real-world death/gore (not anything fictional, even if it's outrageously gory like the head-splosion scene from Scanners, for example, that's all fine).

I'd absolutely love an ability to auto-skip or know about that in a video (for mental-diet reasons), but I would limit it to a small number of *very specific and concrete categories* and avoid feature creep or anything open to interpretation (i.e. "unsettling" or "sexual" are far, far too broad).

**Comment by Madis0** [comment]
What would the difference be between YouTube's own content restrictions?

**Comment by Madis0** [comment]
Okay. Then a bold disclaimer would be needed not to use this for content that actually goes against YouTube's rules.

**Comment by RubenKelevra** [comment]
I mean, content that goes against YouTube rules will just disappear from the platform.

**Comment by nikkehtine** [comment]
Yes, if there's content that goes against YouTube's ToS you should report it to YT right away, not just flag it on sponsorblock lmao

**Comment by aphix** [comment]
> I mean, content that goes against YouTube rules will just disappear from the platform.

Absolutely false. If it is newsworthy I stays and beyond that, honestly, we're faster.

**Comment by post-rex** [comment]
> What would the difference be between this and YouTube's own content restrictions? If anything, Sponsorblock could just indicate adult content (marked by video author on YouTube) similar to the way full-video-segments work, but I don't think it should have a way of censorship by itself.

I would just use it as warning honestly. Anything that might disturb trough one or another way like gory stuff, maybe ear rape or just anything alike.
I don't think it would be censorship.

Knowing something disturbing may happen gives you the ability to prepare for it, and if it was a false flag then nothing is lost.
